dll/directx/wine/ddraw/utils.c

```diff
@@ -580,8 +580,17 @@ unsigned int wined3dmapflags_from_ddrawmapflags(unsigned int flags)
         wined3d_flags |= WINED3D_MAP_NO_DIRTY_UPDATE;
     flags &= ~(handled | DDLOCK_WAIT | DDLOCK_READONLY | DDLOCK_NODIRTYUPDATE);
 
+#ifndef __REACTOS__
     if (flags)
         FIXME("Unhandled flags %#x.\n", flags);
+#elif defined(__REACTOS__) && defined(DBG)
+    if (flags == 0x20)
+    {
+        static int bWarnedOnce = 0; if (!bWarnedOnce) { bWarnedOnce++; FIXME("Unhandled flag 0x20 once\n"); }
+    }
+    else if (flags)
+        FIXME("Unhandled flags %#x.\n", flags);
```

Comment:
You might reorder to stop at `else` and reuse Wine lines.

Reply:
you mean to introduce linebreak after else? I considered that myself also, but then still the indentation wouldn't match the Wine-branch. So I decided to stick to the more structured block here.

Reply:
superseded